agent/src/agent/pipeline/json_builder/json_schema_definitions/observium.json

```diff
@@ -11,6 +11,7 @@
     "dimension_configurations": {"type": "object"},
     "lookup": {"type": "object"},
     "values": {"type": "object"},
+    "values_configurations": {"type": "object"},
```

Comment:
`value_configurations` is correct in English, and please change it in all places